.gitignore

```diff
@@ -8,3 +8,5 @@ src/META-INF
 
 *.swp
 
+.*
+!/.gitignore
```

Comment:
This should not be listed here.. We want to pay attention to the .gitignore file.. In case we change this file in the future, we want Git to recognize those changes..

Reply:
I checked the files that are being ignored with 
`git ls-files -ci --exclude-standard`
and .gitignore is NOT included in the ignored files (the "!" removes it from the .* set)
so I think this is ok.